core/frontend/src/test/Extension.test.ts

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
i would delete this file if we dont actually have a test

Reply:
Removed.